### PR TITLE
Walfly/new step process ui

### PR DESCRIPTION
**Pull request by walfly**

### CODE REVIEW

File: packages/components/src/Modal.tsx [hunk 1]
```diff
@@ -29,15 +29,20 @@ const ModalOuter = styled<ToggleDisplayEl & TextAlignProps, "div">("div")`
 interface ModalInnerProps {
   width?: number;
   padding?: string;
+  fullScreen?: boolean;
```
Comment (am9u) [inline packages/components/src/Modal.tsx]: Just curious, what's the difference between this and `@joincivil/components/src/FullScreenModal`?

File: packages/dapp/src/components/newsroom/NewsroomManagement.tsx [hunk 2]
```diff
@@ -78,6 +79,7 @@ class NewsroomManagement extends React.Component<NewsroomManagementProps, Newsro
           ProposedArticleIds:
           <ul>
             {this.state.proposedArticleIds.map(id => {
+              console.log(id);
```
Comment (am9u) [inline packages/dapp/src/components/newsroom/NewsroomManagement.tsx]: Extra `console.log`?

File: packages/dapp/src/components/newsroom/NewsroomManagement.tsx [hunk 3]
```diff
@@ -238,11 +240,10 @@ class NewsroomManagement extends React.Component<NewsroomManagementProps, Newsro
     }
     if (newsroom) {
       this.state.compositeSubscription.add(
-        newsroom
-          .revisions()
-          .subscribe((contentHeader: any) =>
-            this.setState({ proposedArticleIds: this.state.proposedArticleIds.push(contentHeader.id) }),
-          ),
+        newsroom.revisions().subscribe((contentHeader: any) => {
+          console.log(contentHeader);
```
Comment (am9u) [inline packages/dapp/src/components/newsroom/NewsroomManagement.tsx]: Extra `console.log`?

File: packages/newsroom-signup/src/CompleteYourProfile.tsx [hunk 4]
```diff
@@ -0,0 +1,489 @@
+// import {
```
Comment (am9u) [inline packages/newsroom-signup/src/CompleteYourProfile.tsx]: Should these commented out files just be removed from the repo?

File: packages/newsroom-signup/src/NewsroomUser.tsx [hunk 5]
```diff
@@ -0,0 +1,361 @@
...
+
+export class NewsroomUserComponent extends React.Component<
+  NewsroomUserProps & DispatchProp<any>,
+  TransactionButtonModalFlowState
+> {
+  constructor(props: NewsroomUserProps & DispatchProp<any>) {
+    super(props);
+    this.state = {};
+  }
+  public renderPreMetamMask(): JSX.Element | null {
```
Comment (am9u) [inline packages/newsroom-signup/src/NewsroomUser.tsx]: Tiny typo? S/b `renderPreMetaMask`?

File: packages/dapp/src/components/Dashboard/ActivityListItem.tsx [hunk 6]
```diff
@@ -83,7 +61,6 @@ class ActivityListItemComponent extends React.Component<
       const buttonTextTuple = this.getButtonText();
       const props = {
         newsroomName: newsroomData.name,
-        charter,
```
Comment (tobek) [inline packages/dapp/src/components/Dashboard/ActivityListItem.tsx]: The changes in this file seem to be unrelated to the PR and I think are reverting recent updates. The removal of charter stuff in particular will definitely break things.

File: packages/newsroom-signup/src/actionCreators.ts [hunk 7]
```diff
@@ -0,0 +1,269 @@
+import { AnyAction } from "redux";
```
Comment (tobek) [inline packages/newsroom-signup/src/actionCreators.ts]: Note to us that there's now probably a lot of cruft in here and store in general that can be removed, though some may be needed for later steps so we can hold on for now.

File: packages/components/src/Modal.tsx (hunk 1 above)
Comment (walfly) [reply packages/components/src/Modal.tsx]: didn't know that existed

**Comment by walfly** [comment]
yes the would prevent you from applying through create newsroom.  could you apply in create newsroom before, i thought you had to go to v1 for that?

**Comment by nickreynolds** [comment]
right now the create newsroom flow in the dapp lets you apply without having to go to v1, if there's some way we could maintain that (even if it means having to set something locally) that'd be great. having to go back to v1 would suck

**Comment by walfly** [comment]
@nickreynolds I'll load the old ui based on like a query param or something

**Comment by tobek** [comment]
Yeah, existing UI is needed to make newsrooms with charters for testing. For now let's have the new UI behind a query param or different route and then when we launch it we can, if necessary, swap them out?

**Comment by walfly** [comment]
@tobek @nickreynolds old ui is back in place, new ui is under `/signupNewsroom`
